gazebo.repos

```diff
@@ -0,0 +1,69 @@
+repositories:
+  ros_gz:
+    type: git
+    url: https://github.com/gazebosim/ros_gz
+    version: ros2
```

Comment:
This one should go in dependencies, when the airspeed sensor will be merge this file potencially can be removed